Update fields.md (#1476)
contao/docs: docs/dev/reference/dca/fields.md

| Key         | Description                                                                                                       |
|:------------|:------------------------------------------------------------------------------------------------------------------|

> | digit       | allows numeric characters only (including full stop [.] and minus [-]). Support decimal commas [,] and convert them automatically. |

Reviewer: ```suggestion
| digit       | allows numeric characters only, including full stop [.], minus [-] and decimal commas [,] (the latter will be converted automatically) |
```
